cfme/tests/services/test_provision_stack.py

```diff
@@ -181,35 +205,45 @@ def create_template(setup_provider, provider, dialog):
 
 
 def prepare_stack_data(provider, provisioning):
-    stackname = "test" + fauxfactory.gen_alphanumeric()
+    stack_name = "test" + fauxfactory.gen_alphanumeric()
```

Comment:
There is a nice new vm_name generator we should use here, https://github.com/ManageIQ/integration_tests/blob/f76ef21497a4a3562885d66415902241a471c3da/utils/generators.py#L6